public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,109 @@
+# metrics diff
+
+Find and print changes in [metrics](/doc/command-reference/metrics#description)
+between <abbr>project</abbr> versions.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v]
+                        [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref     Git reference from which diff is calculated. If
+            omitted, `HEAD` (latest commit) is used.
+  b_ref     Git reference to which diff is calculated. If omitted,
+            the current workspace is used instead.
+```
+
+## Description
+
+Calculates the numeric difference (delta) between a metric's value in two
```

Comment:
description is kinda repetitive and does not provide much additional value as it is done now. I would say we should, for example, mention that JSON is flattened, if we support arrays, how do we deal with strange values, etc?

Reply:
Original description https://github.com/iterative/dvc.org/pull/933/commits/5f0da08d7635d195e15b7c81c98c7eda7633318c was more generic, as we are not limited to numeric difference, we can actually print new string value, but the `diff` will not be computed, that is true.

Reply:
Agreed. I've reviewed this to remove unnecessary repetition between short/long desc as well as between desc and options section. I also clarified that it's not just the numeric diff.

> mention that JSON is flattened, if we support arrays, how do we deal with strange values, etc?

I need to think about these ⏳